### PR TITLE
Feature/get transactions executor

### PR DESCRIPTION
**Pull request by motxx**
## What is this pull request?
- Add missing executor of GetTransactions and its integration test
   
## Why do you implement it? Why do we need this pull request?
- Lack of the executor, this API won't work.
- To verify the transaction can be definitely sent.

## Details/Features
- Add executor of GetTransactions
- Add integration test of GetTransactions

## P.S. (optional)
- Current integration tests will be reworked by shared_model later. (IR-715)

### CODE REVIEW

File: schema/queries.proto [hunk 1]
```diff
@@ -21,7 +21,7 @@ message GetAccountAssetTransactions {
 }
 
 message GetTransactions {
-  repeated string tx_hashes = 1;
+  repeated bytes tx_hashes = 1;
```
Comment (l4l) [inline schema/queries.proto]: Why bytes? Quite inconsistent with the rest hash representation

File: test/integration/pipeline/tx_pipeline_integration_test.cpp [hunk 2]
```diff
@@ -15,7 +15,16 @@
  * limitations under the License.
  */
 
+#include <responses.pb.h>
```
Comment (l4l) [inline test/integration/pipeline/tx_pipeline_integration_test.cpp]: In `"` plz

File: test/integration/pipeline/tx_pipeline_integration_test.cpp [hunk 3]
```diff
@@ -15,7 +15,16 @@
...
+#include "datetime/time.hpp"
 #include "integration/pipeline/tx_pipeline_integration_test_fixture.hpp"
+#include "model/generators/query_generator.hpp"
+
+using iroha::model::generators::CommandGenerator;
```
Comment (l4l) [inline test/integration/pipeline/tx_pipeline_integration_test.cpp]: what's the problem with `using namespace ...`?

File: test/integration/pipeline/tx_pipeline_integration_test.cpp [hunk 4]
```diff
@@ -87,3 +97,45 @@ TEST_F(TxPipelineIntegrationTest, TxPipelineTest) {
...
+        iroha::time::now(), CREATOR_ACCOUNT_ID, 1, {given_tx_hash});
+  provider.sign(*query);
+
+  const auto pb_query = PbQueryFactory{}.serialize(query);
+  ASSERT_TRUE(pb_query.has_value());
+
+  iroha::protocol::QueryResponse response;
+  irohad->getQueryService()->FindAsync(pb_query.value(), response);
+  ASSERT_FALSE(response.transactions_response().transactions().empty());
+  const auto got_pb_tx = response.transactions_response().transactions()[0];
```
Comment (l4l) [inline test/integration/pipeline/tx_pipeline_integration_test.cpp]: Check for its size as well, I guess it should be 1

File: schema/queries.proto (hunk 1 above)
Comment (motxx) [reply schema/queries.proto]: Followings are using `bytes`. In Iroha, Blob is used to express pubkey, signature, hash and they all are using `bytes`.
https://github.com/hyperledger/iroha/blob/develop/schema/endpoint.proto#L21-L28
https://github.com/hyperledger/iroha/blob/develop/schema/responses.proto#L83

**Comment by motxx** [comment]
Commit of perms reordering is already discarded.

File: irohad/model/generators/impl/query_generator.cpp [hunk 5]
```diff
@@ -57,7 +57,22 @@ namespace iroha {
...
+        query->creator_account_id = creator;
+        query->query_counter = query_counter;
+        query->account_id = account_id;
+        query->creator_account_id = creator_account_id;
+        query->detail = detail;
+        return query;
+      }
+
+
+        std::shared_ptr<GetSignatories> QueryGenerator::generateGetSignatories(
```
Comment (l4l) [inline irohad/model/generators/impl/query_generator.cpp]: Didn't notice it first time, wrong indent here

File: irohad/model/permissions.hpp [hunk 6]
```diff
@@ -54,8 +54,11 @@ namespace iroha {
     const std::string can_get_domain_signatories = "CanGetDomainSignatories";
 
     const std::string can_get_my_acc_ast = "CanGetMyAccountAssets";
-    const std::string can_get_all_acc_ast = "CanGetAllAccountAssets";
+      const std::string can_get_all_acc_ast = "CanGetAllAccountAssets";
```
Comment (l4l) [inline irohad/model/permissions.hpp]: Same here and below

**Pull request merged**